content/blog/2025/03/security-in-esp-rainmaker/index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "Security in ESP RainMaker"
+date: 2025-03-18
```

Comment:
Thank you @shahpiyushv for the contribution! Please don't forget to update the date.

Reply:
Changed to 2025-03-20